projects/fishtornado_xr/Flocking.cpp

```diff
@@ -318,7 +318,7 @@ void Flocking::Shutdown()
     mMaterialConstants.Destroy();
 }
 
-void Flocking::Update(uint32_t frameIndex)
+void Flocking::Render(uint32_t frameIndex)
```

Comment:
it would be better to keep the original name `Update` instead of renaming it to `Render`. Normally, `Render` would contain Drawcalls.

Reply:
maybe we could pass in `currentViewIndex` into the `Update()` function and update the perFrame and perEye data accordingly?

Reply:
Agree that we should keep the name as `Update`.
I think we don't currently have the perEye/perView data, it is just reusing the same perFrame data, right? and it is indirectly getting the currentViewIndex info from GetCamera() and GetPoseForCurrentView() functions below. 
We can change the whole project to have separate per view data, but probably out scope for this change.

Reply:
Reverted back to the original function names and added the currentViewIndex logic to the Shark's update function. Is the current change what you were looking for @wangra-google @apazylbe ?

Reply:
Flocking, Shark, and Ocean all use time, which is updated in `UpdateTime()`. Perhaps we should be calling `UpdateTime` for view = 0 only, and leave the other `Update` functions the same, so that they use the same time value for all views?

Reply:
That makes sense. Wrapped the UpdateTime call in a conditional

Reply:
@apazylbe I am wondering if we should have a single virtual FrameUpdate() function that is called once per frame not per eye (in application.cpp before the per eye dispatch render), and put any per frame update (like UpdateTime) in there. I think we could have more and more cases that requires update once per frame.

Reply:
@wangra-google Perhaps. I actually think application.cpp shouldn't be calling Render() twice (still need to make that change though), then individual projects can structure Render() as they want. Although "PreRender/FrameUpdate" might be a pretty common method that we might want to add in Application.
No change for this PR though I think.